### DISCUSSION TITLE
Welcome to svelte-jsoneditor Discussions!

### DISCUSSION THREAD

**Discussion by josdejong** [josdejong/svelte-jsoneditor#126] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bestkolobok** [comment]
Hi!

First of all, thank you so much for this great new Jsoneditor. 

Based on it, I created a plugin [vue3-jsoneditor](https://github.com/bestkolobok/vue3-jsoneditor) for Vue3, which, in addition to facilitating integration with the Vue ecosystem, provides the following benefits:
- SSR support
- Full width button
- Automatic dynamic import of query languages
- Automatic dynamic import of dark theme

it would be great if you could include a link to this plugin in the section [wrapper-libraries-for-specific-frameworks](https://github.com/josdejong/svelte-jsoneditor#wrapper-libraries-for-specific-frameworks)

**Reply by josdejong** [reply]
Nice, it looks easy to use and well documented 😎. Thanks for your efforts Alex! I've added your library to the list.

**Reply by bestkolobok** [reply]
Thank you so much, Jos)))

**Comment by WFDexter** [comment]
There doesn't seem to be a great place to do this but this seems like a harmless place....

 I just want to say THANK YOU both for providing this tool in general and specifically for adding the table view and CSV exports recently. That saved me SO much time yesterday trying to figure out what was going on with some data.

**Reply by josdejong** [reply]
Thanks @WFDexter your feedback means a lot to me! 😄
